scripts/normalize-by-median.py

```diff
@@ -55,14 +55,25 @@ def WithDiagnostics(ifile, batch_size, fp, paired, norm):
 
     index = 0
 
-    for index, batch in enumerate(batchwise(
-                                  screed.open(ifile, parse_description=False),
-                                  batch_size)):
+    screed_iter = screed.open(ifile, parse_description=False)
+    reader = broken_paired_reader(screed_iter, force_single=single)
 
-        norm.total += batch_size
+    for index, paired_reads, read0, read1 in reader:
+
+        batch = []
```

Comment:
I'm split on this--Currently, I have it set up so that the actual normalize algorithm/implementation doesn't have to be changed, it still uses a `batch` and not two independent reads (or one read, as the case may be).

I feel like it's cleaner this way--otherwise I'd have to overhaul a lot of Normalizer's logic and it'd just look messier for no real gain in functionality. 

@luizirber @ctb @camillescott Opinions?